external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -2,13 +2,39 @@
 import fsspec
 import yaml
 import os
-from typing import Optional, Union, Sequence, List
+from typing import Dict, Optional, Union, Sequence, List
 
 
 DELP = "pressure_thickness_of_atmospheric_layer"
 MODEL_CONFIG_FILENAME = "training_config.yml"
 
 
+KERAS_MODELS: Dict[str, type] = {}
+SKLEARN_MODEL_TYPES = ["sklearn", "rf", "random_forest", "sklearn_random_forest"]
+
+
+def get_keras_model(name):
+    return KERAS_MODELS[name]
+
+
+def register_keras_trainer(name: str):
```

Comment:
"trainer"  is a synonym of "estimator". Since the latter is an existing convention should you use that instead?

Reply:
OK, I can do that for now. I didn't worry about this too much because I'm definitely renaming "estimator" soon, which is a synonym of "predictor".

Reply:
Huh? Estimate and predict are not synonyms. Predictors only predict, they don't have to be estimated/trained based on some dataset.

Reply:
Huh? Why do you say `estimated/trained`, those aren't synonyms? Are you saying that "estimator" means it's estimating what the model should look like? One can equally take the class name to mean that it "estimates" the tendencies, and that's how I've understood the name up until now.

Thesaurus.com puts "predict" as one of the most relevant synonyms of "estimate". Googling "predict synonyms" and "estimate synonyms" says they're synonyms of one another.

Reply:
Maybe trainer and predictor is clearer. I always associated "estimation" with estimating the parameters of some model (aka training/learning), but I guess the trained/estimated model also "estimates" the predictands.